XU-TechnicalPaper.md

```diff
@@ -25,163 +25,176 @@
   4.2. [Blockstream's Liquid](#42-blockstreams-liquid)  
   4.3. [Lightning/Raiden](#43-lightning--raiden)  
   4.4. [Alphapoints Remarketer](#44-alphapoints-remarketer)  
-  4.5. [Altcoin.io/Etherdelta/Airswap/0x](#45-altcoinio--etherdelta--airswap--0x)  
-  4.6. [RaidEX.io](#46-raidexio)  
-5. [Requirements](#5-requirements-overview)
+  4.5. [Altcoin.io/Etherdelta/Airswap/0x](#45-altcoinio--etherdelta--airswap--0x--idex)  
+  4.6. [sparkswap](#46-sparkswap)  
+  4.7. [RaidEX.io](#47-raidexio)  
+  4.8. [Waves](#48-waves) 
+
 
 ## 1. Executive Summary
 
 1.1. The Vision
 ---------------
-Exchange Union (XU) connects digital asset exchanges by forming **a decentralized network**, which enables **instant and trustless** trades between digital asset exchanges. It enables cross-exchange trading for new trading pairs, access to the best price, and increased liquidity. Read the initial [White Paper](https://api.exchangeunion.com/pdf/en/ExchangeUnion-WhitePaper.pdf) which describes the idea and its use cases.
+Exchange Union (XU) connects digital asset exchanges by forming **a decentralized network**, which enables **instant and trustless** trades between digital asset exchanges, as well as individual traders. It enables cross-exchange trading for increased liquidity, access to the best price and an additional revenue stream for liquidity providers.
```

Comment:
Don't need the commas after "decentralized network" or before "as well as individual traders".

Also, I have been capitalizing it as `xud` instead of `XUD` - at least that's how it is on the main repo. I don't have too strong a preference, but I'd rather stick to one way and lower case is also in line with `lnd` and `btcd`

Reply:
> Don't need the commas after "decentralized network" or before "as well as individual traders".

Done.

> Also, I have been capitalizing it as `xud` instead of `XUD`

Right. I checked it before actually and in non-tech texts LND is usually capitalized. Since we are somewhere in between here, I went ahead and replaced all XUD with `xud`.